routes/admin.php

```diff
@@ -100,7 +104,7 @@
 
 Route::group(['prefix' => 'generic'], function () {
     Route::module('genericPages');
-    Route::module('pages');
+    Route::module('landingPages');
```

Comment:
Define landing pages here seems to duplicate the addition made above. I think it makes sense to live under Generic, so include both `landingPages` and `pages` here.